usaspending_api/search/v2/views/search.py

```diff
@@ -81,20 +84,28 @@ def post(self, request):
         # list of time_period objects ie {"fy": "2017", "quarter": "3"} : 1000
         group_results = OrderedDict()
 
-        # for Subawards we extract data from action_date, for Awards we use sum_transaction_amount
+        # for Subawards we extract data from action_date
         if subawards:
-            data_set = queryset.values('award_type'). \
-                annotate(month=ExtractMonth('action_date'), year=ExtractYear('action_date'),
-                         transaction_amount=Sum('amount')). \
-                values('month', 'year', 'transaction_amount')
+            data_set = queryset \
+                .values('award_type') \
+                .annotate(
+                    month=ExtractMonth('action_date'),
+                    year=ExtractYear('action_date'),
+                    transaction_amount=Sum('amount')) \
+                .values('month', 'year', 'transaction_amount')
         else:
-            data_set = queryset.values('fiscal_year')
-            if not (group == 'fy' or group == 'fiscal_year'):
+            queryset = queryset.values('fiscal_year')
```

Comment:
Can we include the `for Awards we use sum_transaction_amount` down here?

Reply:
What is `sum_transaction_amount`?

Reply:
Sorry, this should just be `#  for Awards we use transaction_amount`.

Reply:
Transaction amount you can see below on lines 100 and 107.